lib/maropost/api.rb

```diff
@@ -53,6 +53,16 @@ def update(contact)
         contact
       end
 
+      def change_email(old_email, new_email)
```

Comment:
should this be returning something better that nil if it can not find the contact by the old email? like a nil contact with errors?
or is the nature of the gem is find already returning nil?

Reply:
Find returns nil if no contact is found. Gatekeeper will not do anything with the result of the change_email method.